documentation/docs/identus/cloud-agent/authentication.md

```diff
@@ -108,24 +108,24 @@ processes.
 Keycloak supports various identity protocols, including OAuth 2.0 and OpenID Connect, making it a versatile solution for
 securing modern web applications.
 
-The PRISM Cloud Agent utilizes the following Keycloak features:
+The Cloud Agent utilizes the following Keycloak features:
 
 - Authentication with JWT Token with a configured flow according to
-  the [ADR](https://staging-docs.atalaprism.io/adrs/adr/20230527-use-keycloak-and-jwt-tokens-for-authentication-and-authorisation-to-facilitate-multitenancy-in-cloud-agent/)
+  the [ADR](https://staging-docs.github.com/hyperledger/identus/adrs/adr/20230527-use-keycloak-and-jwt-tokens-for-authentication-and-authorisation-to-facilitate-multitenancy-in-cloud-agent/)
```

Comment:
I am not sure this link will work. For the moment, the doc website is still unchanged. Let's do the renaming within the website and we will migrate the website after.

Reply:
@CryptoKnightIOG, it's ok to keep the values of the web references unchanged if we don't have the new ones.
So, we need to revert the changes @mkbreuningIOHK pointed to.

Reply:
Fixed